### PR TITLE
Release 0.28.1

### PR DESCRIPTION
**Pull request by n8chur**
# PR Changes

- Bump version to 0.28.1
- `bepi` SampleApp
- Update changelog

# Version Changelog

### Added

- View-backed elements may opt-in to a frame rounding behavior that prioritizes preserving the frame size rather than the frame edges. This is primarily meant for text labels, to fix an issue where labels gain or lose a pixel in rounding and become wrapped or truncated incorrectly. ([#257])

### Changed

- Lifecycle hooks are guaranteed to run after all views are updated. ([#260])

### CODE REVIEW

**Review by kyleve — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +25,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Past Releases
 
+## [0.28.1] - 2021-09-10
```
Comment (kyleve) [inline CHANGELOG.md:28]: Don't forget to add this to the bottom of the file too!

Comment (n8chur) [reply CHANGELOG.md:28]: - [ ] Add version at bottom of changelog

Comment (n8chur) [reply CHANGELOG.md:28]: @kyleve: [This look right?](https://github.com/square/Blueprint/commit/e8cee52bff78d816580d49fea298bffc096ff199)

Comment (kyleve) [reply CHANGELOG.md:28]: Yep!

**Pull request merged**